docs-source/usersguide/application.rst

```diff
@@ -1329,6 +1329,60 @@ separated by spaces instead of commas:
     simulation.reporters.append(StateDataReporter('data.txt', 1000, progress=True,
             temperature=True, totalSteps=10000, separator=' '))
 
+
+Saving Simulation Progress and Results
+==========================================
+
+There are three built-in ways to save the results of your simulation in OpenMM
+(additional methods can be written yourself or imported through other packages
+like mdtraj or parmed). If you are simply interested in saving the structure,
+you can write it out as a PDB file using :code:`PDBFile.writeFile()`.  You can
+see an example of this in the modeller section :ref:`saving-the-results`.
+
+If you are hoping to save more information than just positions, you can use
+:code:`simulation.saveState()`. This will save the entire state of the
+simulation, including positions, velocities, box dimensions and much more in an
+XML file. This same file can be loaded back into OpenMM and used to continue
+the simulation. Importantly, because this file is a text file, it can be
+transfered between different platforms. A potential downside to this approach
```

Comment:
transferred between different platforms *and* different versions of OpenMM, if that is desired.